servers/visual/rasterizer_rd/rasterizer_scene_high_end_rd.cpp

```diff
@@ -809,20 +808,18 @@ void RasterizerSceneHighEndRD::_render_list(RenderingDevice::DrawListID p_draw_l
 			}
 		}
 
-		ShaderVersion shader_version;
+		ShaderVersion shader_version = SHADER_VERSION_MAX; // Assigned to silence wrong -Wmaybe-initialized.
```

Comment:
Same as for roughness, this should never be used uninitialized unless `p_pass_mode` could be given an int value outside the enum.